apps/fluent-tester/src/TestComponents/Menu/consts.ts

```diff
@@ -16,3 +16,8 @@ export const MENU_ON_OPEN = 'Menu_On_Open'; // For testing the open functionalit
 export const MENU_ON_CLOSE = 'Menu_On_Close'; // For testing the open functionality of Menu
 
 export const MENU_DEFOCUS_BUTTON = 'Menu_Defocus_Button'; // Used to move focus away from menu button
+
+export const enum ExpandCollapseState {
```

Comment:
Would it make sense to share the ExpandCollapseState values between all test components?

Reply:
Agreed with Pat. This should probably go in common/consts

Reply:
+1